### PR TITLE
USHIFT-2080: MicroShift Cluster-bot: Skip dnf update if possible

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

**Comment by pmtk** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-clusterbot

File: ci-operator/step-registry/openshift/microshift/clusterbot/prepare-host/openshift-microshift-clusterbot-prepare-host-commands.sh [hunk 1]
```diff
@@ -53,8 +53,13 @@ if [[ -z "${MICROSHIFT_GIT+x}" ]] || [[ "${MICROSHIFT_GIT}" == "" ]]; then
 
 	git clone https://github.com/openshift/microshift -b "release-${OCP_VERSION}" ~/microshift
 
+	configure_args=""
+	if grep -q "\--skip-dnf-update" ~/microshift/scripts/devenv-builder/configure-vm.sh; then
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/clusterbot/prepare-host/openshift-microshift-clusterbot-prepare-host-commands.sh:57]:
```suggestion
	if ~/microshift/scripts/devenv-builder/configure-vm.sh --help | grep -q -- "--skip-dnf-update" ; then
```

File: ci-operator/step-registry/openshift/microshift/clusterbot/prepare-host/openshift-microshift-clusterbot-prepare-host-commands.sh [hunk 2]
```diff
@@ -89,7 +94,12 @@ else
 	: MICROSHIFT_GIT is set - clone it, build it, run it
 
 	git clone https://github.com/openshift/microshift -b "${MICROSHIFT_GIT}" ~/microshift
-	bash -x ~/microshift/scripts/devenv-builder/configure-vm.sh --force-firewall /tmp/pull-secret
+
+	configure_args=""
+	if grep -q "\--skip-dnf-update" ~/microshift/scripts/devenv-builder/configure-vm.sh; then
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/clusterbot/prepare-host/openshift-microshift-clusterbot-prepare-host-commands.sh:99]:
```suggestion
	if ~/microshift/scripts/devenv-builder/configure-vm.sh --help | grep -q -- "--skip-dnf-update" ; then
```

**Comment by pmtk** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-clusterbot

**Pull request merged**